### ISSUE TITLE
setLocalValues on SharpFormAutocompleteField not working with collection of models

### ISSUE THREAD

**Issue by nsrosenqvist** [code16/sharp#156]
The `SharpFormAutocompleteField` documentation specifically states that `setLocalValues` requires an object array, however, the tutorial blog post (https://medium.com/code16/about-sharp-for-laravel-part-2-9c7779782f31) shows `all` being called directly on the model while it doesn't work for me without first converting it to an array: `Users::all()->toArray()`. @dvlpp mentioned that this might be a regression and that I should file it: https://github.com/code16/sharp/issues/153#issuecomment-516344581

```php
    function buildFormFields()
    {
        $this->addField(
            SharpFormAutocompleteListField::make("users")
                ->setLabel("Users")
                ->setRemovable()
                ->setAddable()
                ->setMaxItemCount(User::all()->count())
                ->setAddText("Add User")
                ->setItemField(
                    SharpFormAutocompleteField::make("id", "local")
                        ->setLabel("User")
                        ->setLocalSearchKeys(["name", "email"])
                        ->setLocalValues(User::all()->toArray())
                        ->setListItemInlineTemplate("{{name}} <em>({{email}})</em>")
                        ->setResultItemInlineTemplate("{{name}} <em>({{email}})</em>")
                )
        );
    }
```

**Comment by dvlpp** [comment]
Addressed by #165

**Comment by dvlpp** [comment]
Should be fixed by #165
